dcos_installer/upgrade.py

```diff
@@ -71,6 +71,23 @@
     exit 1
 fi
 
+# Check if the node has node/cluster checks and run them
+if [ -f /opt/mesosphere/etc/dcos-3dt-runner-config.json ]; then
```

Comment:
I added a ticket for removing this test and always running checks after we branch 1.10: https://jira.mesosphere.com/browse/DCOS_OSS-1274